aws-sagemaker-hosted-scorer/src/main/java/ai/h2o/mojos/deploy/sagemaker/hosted/ServeController.java

```diff
@@ -0,0 +1,101 @@
+package ai.h2o.mojos.deploy.sagemaker.hosted;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.gson.JsonParser;
+
+@SuppressWarnings("unused")
+@RestController
+@CrossOrigin
+public class ServeController {
```

Comment:
Ideally, I would tend to keep only Sage specific stuff in this class and reuse the shared code for the parsing and such. See: https://github.com/h2oai/dai-deployment-templates/blob/master/local-rest-scorer/src/main/java/ai/h2o/mojos/deploy/local/rest/controller/ModelsApiController.java

May you need some changes on the shared code portions, but we can always do that. Imho that would be better than having many copies of similar request parsing code with minor alterations. Wdyt?